docs/sites/site_theme.md

```diff
@@ -2,120 +2,133 @@
 
 #### Color Selection
+
 (Default, Cozy, The Daily)
 
 The themes Default, Cozy, and The Daily include an option to enter values for the *main accent color*; Cozy also has an option for the *navigation background color*.
 
 ![color option fields](../sites/sitesfiles/sitetheme_color.png)
 
 The Main Accent color is used for link text and on-hover colors, as in the image below: 
+
 ![arrow points to instances of main accent color in use](../sites/sitesfiles/sitetheme_mainaccent.png)
 
 The navigation background color sets the color of the navigation sidebar for the theme. 
+
 ![arrow points to sidebar](../sites/sitesfiles/sitetheme_navcolor.png)
 
-Edit either field by typing a [color value](https://en.wikipedia.org/wiki/Web_colors) into the field. Either a triplet or a six-character hexadecimal code will work. Once you have entered a valid value, the color bar on the left will automatically update to show the color entered, even without saving changes.
+Edit either field by typing a color value into the field. Either a triplet or a six-character hexadecimal code will work. Once you have entered a valid value, the color bar on the left will automatically update to show the color entered, even without saving changes.
 
-The default values are stored in the prompt text for the field. For the Cozy theme, the default values are: main accent color is `087b94` and navigation background color is `bfdcdc`'
+The default values are stored in the prompt text for the field. For the Cozy theme, the default values are: the main accent color is `087b94` and the navigation background color is `bfdcdc`.
+
+### Layout for browse pages 
 
-### Layout for Browse Pages
 (Center Row, Foundation)
+
 Select how to display items within their browse views.
 
 * **Grid**: Items are organized into rows and columns. This is recommended for items that prominently feature images.
 * **List**: Items are stacked into a single column.
-* **Toggle (default: grid):** Site visitors can choose to display the browse views as grids or lists, with grids as the default.
-* **Toggle (default: list):** Site visitors can choose to display the browse views as grids or lists, with lists as the default.
+* **Toggle (default: grid)**: Site visitors can choose to display the browse views as grids or lists, with grids as the default.
+* **Toggle (default: list)**: Site visitors can choose to display the browse views as grids or lists, with lists as the default.
 
-#### Foundation Theme Settings
-Foundation is theme which comes with a few default stylesheets for fast prototyping or getting started with modifying themes. As a result it has some unique settings. For more information on using Foundation, including working with Sass, see the [theme ReadMe file](https://github.com/omeka-s-themes/foundation-s/blob/master/README.md).
+#### Foundation theme settings
+
+<!--- can the Foundation-specific info just go to its ReadMe page and not be here? --->
```

Comment:
I actually think that the themes need a section of their own like the modules and each should have a page.